src/ArgumentException.php

```diff
@@ -37,37 +34,72 @@ public function __construct(\ReflectionFunctionAbstract $reflection, string $par
         parent::__construct(sprintf((string)static::EXCEPTION_MESSAGE, $parameter, $method, $fileAndLine));
```

Comment:
```suggestion
        parent::__construct(sprintf(static::EXCEPTION_MESSAGE, $parameter, $method, $fileAndLine));
```

Reply:
In this case IDE thinks that is `mixed` because this constant is not final and can be redeclared in a child